zenoh/src/main/java/org/eclipse/zenoh/Workspace.java

```diff
@@ -72,6 +73,40 @@ private Selector toAsbsoluteSelector(Selector s) {
         }
     }
 
+    /**
+     * Put a path/value into Zenoh.
+     *
+     * @param path  the string representing the path. Can be absolute or relative to the workspace.
+     * @param value the value {@link java.lang.String}.
+     * @throws ZException if put failed.
+     */
+    public void put(String path, String value) throws ZException {
+        this.put(new Path(path), new StringValue(value));
```

Comment:
We could avoid boxing of value within a StringValue replacing this line with a code similar to put(String, Value) implementation (i.e. directly calling session.write() with STRING encoding).
For this we would need to add a private static StringValue.encode(String) method.

Reply:
Indeed, that would avoid creating the XYZValue.